pimd/pim_bfd.c

```diff
@@ -142,205 +106,27 @@ static void pim_bfd_reg_dereg_nbr(struct pim_neighbor *nbr, int command)
  *                              zebra for starting/stopping the monitoring of
  *                              the neighbor rechahability.
  */
-int pim_bfd_reg_dereg_all_nbr(struct interface *ifp, int command)
+void pim_bfd_reg_dereg_all_nbr(struct interface *ifp, bool enable)
```

Comment:
Do we actually need the `enable` argument?
I think we can use `pim_ifp->bfd_config.enabled` instead.

Reply:
Good idea, fixed.